lib/App.js

```diff
@@ -2112,15 +2156,36 @@ class App {
       }
     }
 
+    /*
+    * If typescript is used, then add the types for node and add a tsconfig.json file 
+    */
+    if (answers.typescript) {
+      const tsConfigJson = {
+        "compilerOptions": {
+            "module": "commonjs",
+            "target": "ES6",
+            "rootDir": "",
+            "outDir": "build/",
+            "moduleResolution": "node",
+            "esModuleInterop": true,
+            "removeComments": true,
+            "sourceMap": true,
+            "strict": true,
```

Comment:
I am not very confident with these options, are these final?

Reply:
I suspect @tjallingt knows most about this

Reply:
I think for most options here we can use the Node 12 tsconfig base: https://github.com/tsconfig/bases#node-12-tsconfigjson